athena/decode_main.py

```diff
@@ -29,41 +30,23 @@
 )
 
 
-def decode(jsonfile, n=1, log_file=None):
+def decode(jsonfile):
     """ entry point for model decoding, do some preparation work """
     p, model, _, checkpointer = build_model_from_jsonfile(jsonfile)
+    if 'decode_log' in p.decode_config:
+        decode_log = p.decode_config['decode_log']
+        log_dir = os.path.dirname(decode_log)
+        log_name = os.path.basename(decode_log)
+        logging.get_absl_handler().use_absl_log_file(log_name, log_dir)
+    mode = 'avg'
+    if 'load_mode' in p.decode_config:
```

Comment:
We probably should add default values of 'load_mode' and 'decode_log' in decode_config.